<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageEmulatorWindow</class>
 <widget class="QWidget" name="MessageEmulatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSpinBox" name="byte1Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="byte2Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="byte3Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="byte4Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="byte5Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="byte6Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="byte7Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="byte8Spin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="messageLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="periodSpin">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="messageUpdateCheckBox">
       <property name="text">
        <string>Update</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
